### PR TITLE
Mnch 1126 whatsapp list items with comma

### PR DESCRIPTION
**Pull request by Buhle79**
## Purpose
_Describe the problem or feature in addition to a link to the issues._
List items title with the comma import incorrectly

## Approach
_How does this change address the problem?_
remove split on export and import and replaced it with csv memory.

#### Open Questions and Pre-Merge TODOs
- [ ] Use github checklists. When solved, check the box and explain the answer.

## Learning
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

#### Blog Posts
- [How to Pull Request](https://github.com/flexyford/pull-request) Github Repo with Learning focused Pull Request Template.

### CODE REVIEW

File: home/import_content_pages.py [hunk 1]
```diff
@@ -720,3 +721,13 @@ def deserialise_list(value: str) -> list[str]:
     if not value:
         return []
     return [item.strip() for item in value.strip().split(",")]
+
+
+def deserialise_list_with_char(value: str) -> list[str]:
```
[the remark this answers is not in the document]
Comment (Buhle79) [reply home/import_content_pages.py:726]: @rudigiesler I tried this before creating a PR but other tests fails when i merge with the new method of using csv. Should i fix those failing tests or leave these separate functions?

[the remark this answers is not in the document]
Comment (rudigiesler) [reply home/import_content_pages.py:726]: Oh interesting, what would the changes required be? If it's not doing the same thing like I thought, then we'll want to keep it separate.

[the remark this answers is not in the document]
Comment (Buhle79) [reply home/import_content_pages.py:726]: Let me work around these failing tests, i will let update you.

**Comment by Buhle79** [comment]
Hi @rudi this task is ready for re look please.

**Review by rudigiesler — APPROVED** [review]
Looks good!

**Pull request merged**